l10n_ch_payment_slip/i18n/fr_CH.po

```diff
@@ -0,0 +1,432 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_ch_payment_slip
```

Comment:
I added this one so you could use already translated new fields for fr.po
I was not aware of the override.